packages/graph/src/graph2/graph.ts

```diff
@@ -352,7 +352,12 @@ export class Graph2 extends SVGZoomWidget {
             [x, y] = [0, 0]; // center(item.points);
         }
         if (x !== undefined && y !== undefined) {
-            const bbox = item.element.node().getBBox();
+            const node = item.element.node();
+            const bbox = node.getBBox();
+            const xforms = node.getAttribute('transform');
+            const parts  = /translate\(\s*([^\s,)]+)[ ,]([^\s,)]+)/.exec(xforms);
+            x = +parts[1];
+            y = +parts[2];
```

Comment:
I suspect the "project", "projectPlaceholder" and "rproject" methods would be better here (they ultimately are what created the translate that your parsing?